supplementary_style_guide/introduction/about-guide.adoc

```diff
@@ -1,21 +1,28 @@
 = About this guide
 
-This guide provides style guidelines and preferred term usage for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. It is intended to be used as a supplement to the _IBM Style_ guide, which is the primary source of guidance for Red Hat product documentation.
+The style and language guidance in this guide overrides or supplements some guidance provided by _IBM Style_, which is the primary source of style guidance for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. 
```

Comment:
To be consistent with how we reference it through the rest of the guide

```suggestion
The style and language guidance in this guide overrides or supplements some guidance provided by the _IBM Style_ guide, which is the primary source of style guidance for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. 
```

Reply:
Fixed